app/controllers/admin/products_v3_controller.rb

```diff
@@ -68,6 +68,28 @@ def destroy_variant
       end
     end
 
+    def clone
+      @product = Spree::Product.find(params[:id])
```

Comment:
I think we could be using Admin::ResourceController to automatically `load_resource` for us. But that might bring new challenges... let's not bother right now.